sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeEntityAsyncClient.java

```diff
@@ -101,7 +103,7 @@
      * @return A mono {@link Response} that contains {@link RecognizeEntitiesResultCollection}.
      */
     Mono<Response<RecognizeEntitiesResultCollection>> recognizeEntitiesBatch(
-        Iterable<TextDocumentInput> documents, TextAnalyticsRequestOptions options) {
+        Iterable<TextDocumentInput> documents, RecognizeEntitiesOptions options) {
```

Comment:
would it be ok if u generate a API View update with these changes so we can see the new overloads?

Reply:
https://apiview.dev/Assemblies/Review/329cfc783b474d97891a5349e705feb4?diffRevisionId=a4881f7f0089462eb8a71ec4fe700483&doc=False&diffOnly=False

Reply:
Thank you!
So from it, it seems like you are only adding the `RecognizeEntitiesOptions ` to the `xxResponse` methods. is that intended?

What about `recognizeEntitiesBatch` method?

Reply:
For the method that takes List<String> it is fine since it is a overload method. Currently, we only add it to the maxOverload method to replace the `TextAnalyticsRequestOptions` . If in the future, there is a need, we can add it to overload method that takes List<String>. Trying to not introduced too many method that has little impact.